app/views/shared/check_answers/_merits_proceeding_section.html.erb

```diff
@@ -4,7 +4,7 @@
   <section class="print-no-break govuk-!-padding-top-6">
     <h2 class="govuk-heading-l"><%= proceeding.meaning %></h2>
 
-    <%= govuk_summary_list(card: { title: proceeding.meaning }, actions: !read_only) do |summary_list| %>
+    <%= govuk_summary_list(card: { title: t(".vary_order_title") }, actions: !read_only) do |summary_list| %>
```

Comment:
will this not be changing the title for every proceeding?

Reply:
Have test it and it does do that ^^ on the check_merits_answers question. Each of the proceedings i selected have a card title of the "vary order" title

![Screenshot 2024-12-03 at 17 23 12](https://github.com/user-attachments/assets/4a437ddb-ad2c-4ac7-af20-7d52d5c3511a)


I believe you just need to undo this change

Reply:
Thanks for spotting this, @jsugarman. After discussions with you, @agoldstone93 and @kmahern, I have updated the Check Merits Answers page as shown below for now. Katharine will update the Check Merits Answers page to reflect the design in [AP-5584](https://dsdmoj.atlassian.net/browse/AP-5584).

![Screenshot 2024-12-04 at 12 15 50](https://github.com/user-attachments/assets/da5e7e6c-20ea-4baa-9e41-dce1ffed34d9)